modules/rosa-cluster-autoscaler-cli-interactive-during.adoc

```diff
@@ -19,9 +19,10 @@ Interactive mode provides more information about available configurable paramete
 ----
 $ rosa create cluster --cluster-name <cluster_name> --enable-autoscaling --interactive
 ----
+include::snippets/rosa-long-cluster-name.adoc[]
 +
```

Comment:
This is rendering incorrectly in the preview. 
```suggestion

```
![image](https://github.com/openshift/openshift-docs/assets/25209155/eeb0cb14-8162-4c44-8765-cf35956c0397)